Feature: login on chrome # tests/login/login.feature:1

  Background:   # tests/login/login.feature:3

  @positive-case
  Scenario: Login with correct credentials                               # tests/login/login.feature:7
    Given I open the chrome browser                                      # None
    Given I open the login page                                          # None
    When I enter the correct username "hongducdev" and password "123456" # None
    Then I should be logged in successfully                              # None

  @negative-case
  Scenario: Login with incorrect password                                                     # tests/login/login.feature:13
    Given I open the chrome browser                                                           # tests/login/steps/background_steps.py:5
    Given I open the login page                                                               # tests/login/steps/steps.py:5
    When I enter the correct username "hongducdev" and incorrect password "incorrectpassword" # tests/login/steps/steps.py:15
    Then I should not be logged in                                                            # tests/login/steps/steps.py:30
    And I should see the message "Incorrect password"                                         # tests/login/steps/steps.py:35
      Assertion Failed: Expected error message: Incorrect password. Actual error message: Failed to fetch


  @negative-case
  Scenario: Login with nonexistent username                                         # tests/login/login.feature:20
    Given I open the chrome browser                                                 # tests/login/steps/background_steps.py:5
    Given I open the login page                                                     # tests/login/steps/steps.py:5
    When I enter a nonexistent username "nonexistentusername" and password "123456" # tests/login/steps/steps.py:20
    Then I should not be logged in                                                  # tests/login/steps/steps.py:30
    And I should see the message "Nonexistent username"                             # tests/login/steps/steps.py:35
      Assertion Failed: Expected error message: Nonexistent username. Actual error message: Failed to fetch



Failing scenarios:
  tests/login/login.feature:13  Login with incorrect password
  tests/login/login.feature:20  Login with nonexistent username

0 features passed, 1 failed, 0 skipped
0 scenarios passed, 2 failed, 1 skipped
8 steps passed, 2 failed, 4 skipped, 0 undefined
Took 0m3.495s
